android-build/README.md

```diff
@@ -1,18 +1,23 @@
-# Android applications build
+# Android build image
+### `dubit/android-build-image`
 
-## Docker Hub repository: [Link](https://hub.docker.com/r/dubit/android-build-image/)
+This image contains an environment that allows building Android applications within a containerized environment. It is roughly based on the [Bitrise.io Android image](https://github.com/bitrise-docker/android).
+
+## Tools, compilers and utilities inside:
+* Base OS: Ubuntu:16.04
+* Version control: Git
+* System tools: curl, unzip, build-essential
+* Build tools: maven, gradle, react-native CLI
+* Compilers and programming languages: GCC, G++, python, openJDK-8, Node.js 6.x
+* SDKs: Android SDK core components and Android NDK
```

Comment:
Probably yes. The problem is with the native system and build tools since we pull those from Ubuntu's package manager.

We can probably list the Android library versions and everything.